pkgs/applications/networking/p2p/fragments/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchgit
+, meson
+, ninja
+, cmake
+, desktop-file-utils
+, automake
+, autoconf
+, libtool
+, libevent
+, openssl
+, zlib
+, pkgconfig
+, libgee
+, curl
+, vala
+, glib
+, python3
+, gtk3
+, libhandy
+, hicolor-icon-theme
+, libtransmission
+, libb64
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "fragments";
+  version = "1.5";
+
+  src = fetchgit {
+    url = "https://gitlab.gnome.org/World/Fragments";
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "0x1kafhlgyi65l4w67c24r8mpvasg3q3c4wlgnjc9sxvp6ki7xbn";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    cmake
+    desktop-file-utils
+    libtool
+    meson
+    ninja
+    pkgconfig
```

Comment:
You are sure we need all of these?

Reply:
Good point, I'll check this after I get this to compile properly :)